Formula/kubie.rb

```diff
@@ -0,0 +1,43 @@
+class Kubie < Formula
+  desc "Much more powerful alternative to kubectx and kubens"
+  homepage "https://blog.sbstp.ca/introducing-kubie/"
+  url "https://github.com/sbstp/kubie/archive/v0.8.3.tar.gz"
+  sha256 "d8d4fb2e2d2db45fab242dc9af5f6e5e29629c0dfc3a39c95e5f19db63f7b704"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--locked", "--root", prefix, "--path", "."
+    bash_completion.install "./completion/kubie.bash"
+  end
+
+  test do
+    kube_config = [
+      "apiVersion: v1",
+      "clusters:",
+      "- cluster:",
+      "    server: http://0.0.0.0/",
+      "  name: kubie-test-cluster",
+      "contexts:",
+      "- context:",
+      "    cluster: kubie-test-cluster",
+      "    user: kubie-test-user",
+      "    namespace: kubie-test-namespace",
+      "  name: kubie-test",
+      "current-context: baz",
+      "kind: Config",
+      "preferences: {}",
+      "users:",
+      "- user:",
+      "  name: kubie-test-user",
+    ].join("\n") + "\n"
+    kube_config_dir = "#{ENV["HOME"]}/.kube"
+    mkdir_p(kube_config_dir)
+    File.write("#{kube_config_dir}/kubie-test.yaml", kube_config)
```

Comment:
Once again, please use this style to write files.
```suggestion
    mkdir_p(ENV["HOME"]/".kube")
    (ENV["HOME"]/".kube/kubie-test.yaml").write <<~EOS
      apiVersion: v1
      clusters:
      - cluster:
          server: http://0.0.0.0/
        name: kubie-test-cluster
      contexts:
      - context:
          cluster: kubie-test-cluster
          user: kubie-test-user
          namespace: kubie-test-namespace
        name: kubie-test
      current-context: baz
      kind: Config
      preferences: {}
      users:
      - user:
        name: kubie-test-user
    EOS
```